### ISSUE TITLE
graphics memory won't be released?

### ISSUE THREAD

**Issue by astroboylrx** [JohnCoates/Aerial#621]
#### Required information
In order to help us sort your issue, we ask that you provide the following information:
- [ ] Mac model: 2018 MBP 15-inch + i7-8850H + AMD Radeon Pro 560X
- [ ] macOS version: macOS 10.14.1
- [ ] Monitor setup: bulit-in monitor + Dell U2415 (1920x1200) + ASUS VS238 (1920x1080)

If appropriate, please enable `Debug mode` and `Log to disk` in `Advanced` tab and replicate your bug, then attach the `AerialLog.txt` file (You can access this file through the Advanced tab).

[AerialLog.txt](https://github.com/JohnCoates/Aerial/files/2551022/AerialLog.txt)

#### Description of issue / Feature request

I've been using the 4K HEVC mode for Aerial. I can understand it takes a lot of graphics memory for my three monitors. But it seems the graphics memory won't be released afterward (I don't think the log file indicate that) according to istats menu. 
![image](https://user-images.githubusercontent.com/3999966/48033445-8d3f4a00-e118-11e8-9341-0f5f0736a2e7.png)

**Comment by glouel** [comment]
"Graphics memory" is a pretty vague thing, same for gpu usage which is vaguely defined. 

I would check if VTDecoderXPCService is hanging around after you exit screensaver, it's the Graphics Video Acceleration service that is used by the system to decode videos. I've seen it hang around sometimes for a bit before quitting (maybe in case playback needs to be resumed ?) and it may hang to that "memory", whatever that is. 

This is far above Aerial's level though, GVA is completely opaque to us and we are just incidently using it. I don't think there's an issue here, I suggest you keep a look for VTDecoderXPCService and see if, when that service quits, the "memory" lowers ?

**Comment by astroboylrx** [comment]
Thank you so much for the prompt reply and explanations. I indeed found there are four `VTDecoderXPCService` hanging around. Killing those processes did not lower the "memory".  

![image](https://user-images.githubusercontent.com/3999966/48034066-0f307280-e11b-11e8-8e12-8b0c0e6e01ee.png)

I ended up with unplugging my external monitors and replugging them, which eliminated those "memory" usage. I see this is above Aerial's level. I can close this issue. :-)
